src/main/java/de/tum/in/www1/artemis/web/rest/metis/conversation/dtos/ChannelDTO.java

```diff
@@ -0,0 +1,115 @@
+package de.tum.in.www1.artemis.web.rest.metis.conversation.dtos;
+
+import de.tum.in.www1.artemis.domain.metis.conversation.Channel;
+
+public class ChannelDTO extends ConversationDTO {
+
+    private String name;
+
+    private String description;
+
+    private String topic;
+
+    private Boolean isPublic;
+
+    private Boolean isAnnouncementChannel;
+
+    private Boolean isArchived;
+
+    // property not taken from entity
```

Comment:
What does that mean?

Reply:
Tthat it is not taken from the entity but calculated / based on other entities